src/eventmaster/api_handlers.go

```diff
@@ -143,12 +143,12 @@ func (h *httpHandler) sendError(w http.ResponseWriter, code int, err error, mess
 	w.Write([]byte(errMsg))
 }
 
-func (h *httpHandler) sendResp(w http.ResponseWriter, key string, val string, path string) {
+func (h *httpHandler) sendResp(w http.ResponseWriter, key string, val interface{}, path string) {
```

Comment:
Currently `val` can be a `string` or `slice` and they'd both get marshaled into JSON the same way since the type is inferred